docs/developer_guide/compute.md

```diff
@@ -250,7 +250,7 @@ import starsCount from './starsCount'
       showToast(string`The repos have ${starsCount} stars`, 5000, 'success')
     ],
     catch: new Map([
-      [HttpProviderError, showToast(string`Error: ${props`error.message`}`, 5000)]
+      [HttpProviderError, showToast(string`Error: ${props`error.body.message`}`, 5000)]
```

Comment:
you can test it via the bin which is used in the "Router" -part:
https://www.webpackbin.com/bins/-KdBeIDJoRv0PQlF7uWU

but it is wrong anyway...so there is a bin change :)